### DISCUSSION TITLE
Consider outreach to yusancky/setup-typst?

### DISCUSSION THREAD

**Discussion by jcbhmr** [typst-community/org#1] [Ideas]
I recently opened a Pull Request to revamp the setup-typst action https://github.com/yusancky/setup-typst/pull/12 by @yusancky. That got me thinking: I think that https://github.com/yusancky/setup-typst would be a good project to have under the @typst-community umbrella.

Why this would be a good idea:
- Avoid the "what's the right Typst installer action for GitHub Actions" problem https://github.com/rust-lang/rustup/issues/3409
- Centralize the Action so that other @typst-community members can update it if the maintainer loses interest (happens all the time lol). Increase the [bus-factor](https://en.wikipedia.org/wiki/Bus_factor).
- Provide a "pretty name" namespace so it's `typst-community/setup-typst` which looks "official-ish" more than a person's username does lol 😅. That's branding for you haha.

**Comment by jcbhmr** [comment]
There's been some discussion on Discord about what to do:

> Jacob Hummer — Today at 3:37 PM
> how would you like to procceed? i'd very much like to see a typst-community/setup-typst.
> that could be done via:
> working with and improving https://github.com/yusancky/setup-typst and transfering it to typst-community
> forking it (https://github.com/jcbhmr/setup-typst or your own version idc) and moving that to typst-community
> ignoring forking and just create a new version (which is basically when ended up happening in https://github.com/jcbhmr/setup-typst lol)

A good compromise I think would be to use https://github.com/jcbhmr/setup-typst [which currently works](https://github.com/jcbhmr/setup-typst/actions/runs/7240554374) and release it as "typst-community/setup-typst@v3". Then, if/when @yusancky is active (last commit 6mo) we can merge the fork _back into https://github.com/yusancky/setup-typst_ and then transfer that repository to @typst-community ? Does that seem like a good idea?

**Reply by jcbhmr** [reply]
given https://github.com/yusancky/setup-typst/pull/12#issuecomment-1862188566 i dont think there's going to be as much of a communication delay as i might have feared (i feared a multi-week delay lol; a few days is ⚡ quick!)
so ignore ☝☝ the parent comment about suggestion to fork

**Comment by huwaireb** [comment]
Nobody represents the organization, but the community involved. You are completely free to conduct the outreach in the name of `typst-community`, as long as you believe it has a place here. 

However, since the organization itself is still in its preliminary stages, i'd refrain till its setup to eliminate confusion that will arise otherwise.

**Comment by huwaireb** [comment]
Completed with @yusancky joining the Typst Community and a new @typst-community/setup-typst team being created.
